btree/btree.c

```diff
@@ -3239,6 +3238,13 @@ static void btree_tx_commit_cb(void *payload)
 	node_lock(node);
 	M0_ASSERT(node->n_txref != 0);
 	node->n_txref--;
+	if (!node->n_be_node_valid && node->n_txref == 0) {
```

Comment:
You also need to check if (node->n_ref == 0)

Reply:
done.